daemon/api_test.go

```diff
@@ -935,3 +939,102 @@ func (s *apiSuite) TestServiceLogs(c *check.C) {
 		Result: []map[string]interface{}{{"message": "hi", "timestamp": "42", "raw": log}},
 	})
 }
+
+func (s *apiSuite) TestMetaIconGet(c *check.C) {
+	// have an “icon” on the system
+	c.Check(os.MkdirAll(dirs.SnapIconsDir, 0755), check.IsNil)
+	c.Check(ioutil.WriteFile(filepath.Join(dirs.SnapIconsDir, "yadda"), []byte("yadda icon"), 0644), check.IsNil)
```

Comment:
Shouldn't these be assertions? The following probably doesn't make much sense if either of these fail.